frontends/web/src/routes/bitsurance/widget.tsx

```diff
@@ -99,18 +99,50 @@ export const BitsuranceWidget = ({ code }: TProps) => {
     current.contentWindow?.postMessage(message, '*');
   };
 
+  const sendAddressWithXPub = (address: string, sig: string, xpub: string) => {
+    const { current } = iframeRef;
+
+    if (!current) {
+      return;
+    }
+
+    const message = serializeMessage({
+      version: MessageVersion.V0,
+      type: V0MessageType.Address,
+      bitcoinAddress: address,
+      extendedPublicKey: xpub,
+      signature: sig,
+    });
+
+    current.contentWindow?.postMessage(message, '*');
+  };
+
   const handleRequestAddress = (message: RequestAddressV0Message) => {
     signing = true;
     const addressType = message.withScriptType ? String(message.withScriptType) : '';
     const withMessageSignature = message.withMessageSignature ? message.withMessageSignature : '';
+    const withExtendedPublicKey: boolean = message.withExtendedPublicKey ? message.withExtendedPublicKey : false;
```

Comment:
nit: shorter alternative 
```suggestion
    const withExtendedPublicKey = !!message.withExtendedPublicKey;
```

Reply:
That looks much nicer. thx.